runtime-sdk/src/modules/rofl/mod.rs

```diff
@@ -492,6 +492,17 @@ impl<Cfg: Config> Module<Cfg> {
         Self::get_instances(args.id)
     }
 
+    /// Returns the minimum amount of required stake for managing ROFL application.
+    #[handler(query = "rofl.AppStakeThresholds")]
```

Comment:
> unfortunately

There are good reasons why this is not the case by default:
* Queries can be non-deterministic which could cause discrepancies and round failures.
* Queries can be expensive.
* Queries don't usually account for gas which is needed for safe operation from calls.

We could however extend the procedural macros to support an attribute that would expose a specific query as an internal call as well. But this would just save some typing and would still need to be explicit to be safe (with gas accounting handled by the query impl when run in a call context).

Reply:
Yup this was just a note, and a question really about did we want to expose this to solidity

Reply:
I don't see a reason why not to.